src/gate_detect/include/GateDetector.h

```diff
@@ -105,7 +107,9 @@ class GateDetector {
                    int houghLinesMaxLineGap,
                    int poleMax,
                    float interpolationConstant1,
-                   float interpolationConstant2);
+                   float interpolationConstant2,
```

Comment:
What is `interpolationConstant2` and `interpolationConstant1`? We might want to consider changing these names to be more descriptive.

Reply:
Yeah I don't really know what else to call them, I put a description in the header file of what they are. They are just the mapping of pixel width to gate distance which may actually need to be adjusted